web/src/models/Span.model.ts

```diff
@@ -42,7 +42,7 @@ const Span = ({id = '', attributes = {}, startTime = 0, endTime = 0, parentId =
 
   const name = attributes[Attributes.NAME] || '';
   const kind = SpanKindList.find(spanKind => spanKind === attributes[Attributes.KIND]) || SpanKind.INTERNAL;
-  const duration = Number(attributes[Attributes.TRACETEST_SPAN_DURATION]) || 0;
+  const duration = attributes[Attributes.TRACETEST_SPAN_DURATION] || '0ns';
```

Comment:
The duration now is a string. The BE change broke the timeline graph. We need to figure out the best approach  for the duration.